subprojects/docs/src/docs/userguide/tasks.xml

```diff
@@ -290,52 +290,673 @@
     </section>
 
     <section id="sec:up_to_date_checks">
-        <title>Skipping tasks that are up-to-date</title>
-        <para>If you are using one of the tasks that come with Gradle, such as a task added by the Java plugin,
-            you might have noticed that Gradle will skip tasks that are up-to-date. This behaviour is also available
-            for your tasks, not just for built-in tasks.
+        <title>Up-to-date checks (AKA Incremental Build)</title>
+        <para>An important part of any build tool is the ability to avoid doing work that has already
+            been done. Consider the process of compilation. Once your source files have been compiled,
+            there should be no need to recompile them unless something has changed that affects the
+            output, such as the modification of a source file or the removal of an output file. And
+            compilation can take a significant amount of time, so skipping the step when it’s not needed
+            saves a lot of time.
         </para>
+        <para>Gradle supports this behavior out of the box through a feature it calls incremental build.
+            You have almost certainly already seen it in action: it’s active nearly<footnote>
```

Comment:
Ah, no. This is the way that footnotes work in DocBook. "it's active nearly" is followed by " every time the UP-TO-DATE text appears..." below.